packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -153,10 +167,12 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
         minIncludeTextWidth: 1050,
         extensions: [extensions.slowAnimations],
       ),
+      const SizedBox(width: denseSpacing),
```

Comment:
could you make these same changes to the Timeline page for consistency?

Reply:
👍 will do